react/react1/week2/todo-list/src/components/App/App.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from "react";
+import { List } from "../List/List";
+import { Input } from "../Input/Input";
+import { Button } from "../Button/Button";
+import { Timer } from "../Timer/Timer";
+
+
+export default class App extends Component {
```

Comment:
I see you tried class component instead of function components. Nice that you want to try both, but generally try and stick to function components and especially avoid mixing them in the same project

Reply:
sure, I will keep in mind